### DISCUSSION TITLE
Deployment on openstack

### DISCUSSION THREAD

**Discussion by skiptoniam** [magda-io/magda#3415] [Q&A]
Hi,
I was wondering if there was any guidance for deploying on openstack/nectar. Maybe I can use the AWS as a guide?
Regards,
Skip.

**Comment by tanerjn** [comment]
It should be more or less same principles to deploy OpenStack with the components you will need, @skiptoniam

I guess nectar is an example platform funded/developed by Australian government. You can check for a generic deployment example here:

https://github.com/tanerjn/openstack-deployment

There is a 5-page research paper at the end of the markdown.
